ipn/ipnlocal/local.go

```diff
@@ -4791,3 +4798,23 @@ func (b *LocalBackend) ResetAuth() error {
 	}
 	return b.resetForProfileChangeLockedOnEntry()
 }
+
+// StreamDebugCapture writes a pcap stream of packets traversing
+// tailscaled to the provided response writer.
+func (b *LocalBackend) StreamDebugCapture(w http.ResponseWriter) error {
```

Comment:
having a ResponseWriter deep in here feels a bit strange. Could this just be an io.Writer?